oneanddone/tests/functional/tests/test_task_assignment.py

```diff
@@ -22,4 +22,5 @@ def test_that_unassigned_task_is_available(self, base_url, selenium, task, new_u
         home_page.login(new_user)
         available_tasks_page = home_page.click_available_tasks()
         home_page.search_for_task(task.name)
-        assert len(available_tasks_page.available_tasks)
+        available_tasks = x.name for x in available_tasks_page.available_tasks
+        assert available_tasks.count(task.name) == 1
```

Comment:
Aren't the task names unique? Searching for the unique task name should allow us to verify the length of available tasks is 1. This is failing in Travis CI - did you run it locally before pushing?